### PR TITLE
ObjectID rule

### PR DESCRIPTION
**Pull request by erfanium**
For validating mongodb/bson ObjectID. I'll update README.md after approvement

### CODE REVIEW

File: lib/rules/objectID.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+"use strict";
+
+/**	Signature: function(value, field, parent, errors, context)
+ */
+module.exports = function({ schema, messages, index }, path, context) {
+	const src = [];
+
+	if (!context.customs[index]) context.customs[index] = { schema };
+	else context.customs[index].schema = schema;
```
Comment (erfanium) [inline lib/rules/objectID.js:9]: This line is not easy to test, that's why coverage decreased

**Comment by erfanium** [comment]
@icebob  The main advantage of this is we can use it directly in FV schema
```js
params = {
   id: "objectID",
   name: "string",
   age: "number"
}  
```

**Comment by icebob** [comment]
I mean we don't need the mongodb lib for validation. Because it's enough simple. We can copy it into the rule source.

**Comment by erfanium** [comment]
@icebob But what about `convert`? It's very efficient, I use it myself

**Comment by icebob** [comment]
As I see, it just convert the hex string to binary bytes with `Buffer`
https://github.com/williamkapke/bson-objectid/blob/master/objectid.js#L39-L43

**Comment by icebob** [comment]
But yeah, the `instanceof` won't be correct.

**Comment by icebob** [comment]
I've approved this PR. Will you update the README in this PR?

**Comment by erfanium** [comment]
@icebob Yes, I do it whenever I have time, Probably the next two days, sorry for the delay

**Comment by icebob** [comment]
No problem. Take your time, just I thought maybe you forgot it :)

**Pull request merged**